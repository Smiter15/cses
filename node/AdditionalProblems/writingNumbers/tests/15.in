453825315549727902
