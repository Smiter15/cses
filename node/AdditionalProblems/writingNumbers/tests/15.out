208515131670183450
